src/libraries/Microsoft.Extensions.Hosting.Abstractions/ref/Microsoft.Extensions.Hosting.Abstractions.cs

```diff
@@ -116,15 +116,15 @@ public partial interface IHostedService
     }
     public partial interface IHostEnvironment
     {
-        string ApplicationName { get; set; }
+        string? ApplicationName { get; set; }
         Microsoft.Extensions.FileProviders.IFileProvider ContentRootFileProvider { get; set; }
         string ContentRootPath { get; set; }
```

Comment:
We will probably have problems with these two in the `Hosting` PR, but we can deal with it then. I think the intention here is that these two properties are not `null`.